jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ConcurrentPool.java

```diff
@@ -319,6 +340,19 @@ public void dump(Appendable out, String indent) throws IOException
             new DumpableCollection("entries", entries));
     }
 
+    public int sweep()
```

Comment:
We need a good way to automatically initiate a sweep.   Perhaps whenever there is an acquire operation that does not find any available entries, then it should sweep?    Alternately we could always poll a reference queue on acquire and if we ever see a value there then we do a sweep?